actionpack/lib/action_dispatch/middleware/params_parser.rb

```diff
@@ -1,15 +1,16 @@
 require 'active_support/core_ext/hash/conversions'
 require 'action_dispatch/http/request'
 require 'active_support/core_ext/hash/indifferent_access'
+require "active_support/core_ext/module/delegation"
 
 module ActionDispatch
   class ParamsParser
     class ParseError < StandardError
-      attr_reader :original_exception
+      delegate :to_s, to: :cause
 
-      def initialize(message, original_exception)
-        super(message)
-        @original_exception = original_exception
```

Comment:
We'll need to deprecate this method signature, too, since apps may provide custom params parsers.

Using `super message` makes sense as well. Then we don't have to delegate `to_s` (or `message` or …)